go/bind/keybase.go

```diff
@@ -160,7 +160,7 @@ func Init(homeDir string, mobileSharedHome string, logFile string, runModeStr st
 
 	kbCtx = libkb.NewGlobalContext()
 	kbCtx.Init()
-	kbCtx.SetServices(externals.GetServices())
+	kbCtx.SetServices(externals.NewExternalServices(kbCtx))
```

Comment:
while we're doing this refactor can we rename the module? i've always found it a bit confusing... maybe `proof_services`?

Reply:
For the sake of diff readability that should be its own PR. If packages can handle underscores that seems nice